python/ray/dataframe/dataframe.py

```diff
@@ -18,6 +18,7 @@
 import numpy as np
 import ray
 import itertools
+from df_utils import *
```

Comment:
Rather than `import *` could you import the functions used directly? `utils` will be for other files too, and we don't want to unnecessarily import.

Reply:
Done